src/process/ProcessManagerImpl.cpp

```diff
@@ -159,6 +159,41 @@ ProcessManagerImpl::shutdown()
     }
 }
 
+void
+ProcessManagerImpl::shutdownProcess(std::shared_ptr<ProcessExitEvent> pev)
```

Comment:
this implementation of `shutdownProcess` is not desirable in the typical case (it's doing `kill -9 ...`):
we want "clean" shutdown by default in case processes start sub processes. The process hierarchy is typically something like `stellar-core -> bash -> aws_cli-> aws_cli_children`, a "force kill" of `bash` may leave the `aws_cli` process (and children) running.

You can keep this implementation under a "force" parameter, but the MVP may not need this (ie: the only place where we would need it is if we're implementing timeout for abort)